docs/sources/panels-visualizations/visualizations/table/index.md

```diff
@@ -191,6 +191,18 @@ To filter column values, click the filter (funnel) icon next to a column title.
 
 Click the check box next to the values that you want to display. Enter text in the search field at the top to show those values in the display so that you can select them rather than scroll to find them.
 
+### Select column values
```

Comment:
If I'm understanding correctly, this is a function to select or deselect all the visible values and add them to your filtering. 
```suggestion
### Select all displayed column values
```

Reply:
Yes, this is the idea, thanks you very much for your changes!